RetakesAllocatorCore/Config/Configs.cs

```diff
@@ -112,11 +114,35 @@ public record RoundTypeManualOrderingItem(RoundType Type, int Count);
 
 public record ConfigData
 {
-    public List<CsItem> UsableWeapons { get; set; } = WeaponHelpers.GetAllWeapons();
+    public List<CsItem> UsableWeapons { get; set; } = WeaponHelpers.AllWeapons;
 
     public List<WeaponSelectionType> AllowedWeaponSelectionTypes { get; set; } =
         Enum.GetValues<WeaponSelectionType>().ToList();
 
+    public Dictionary<CsTeam, Dictionary<WeaponAllocationType, CsItem>> DefaultWeapons { get; set; } = new()
+    {
+        {
+            CsTeam.Terrorist, new()
+            {
+                {WeaponAllocationType.PistolRound, CsItem.Glock},
+                {WeaponAllocationType.Secondary, CsItem.Deagle},
+                {WeaponAllocationType.HalfBuyPrimary, CsItem.Mac10},
+                {WeaponAllocationType.FullBuyPrimary, CsItem.AK47},
+                {WeaponAllocationType.Preferred, CsItem.AWP},
+            }
+        },
+        {
+            CsTeam.CounterTerrorist, new()
+            {
+                {WeaponAllocationType.PistolRound, CsItem.USPS},
+                {WeaponAllocationType.Secondary, CsItem.Deagle},
+                {WeaponAllocationType.HalfBuyPrimary, CsItem.MP9},
+                {WeaponAllocationType.FullBuyPrimary, CsItem.M4A4},
+                {WeaponAllocationType.Preferred, CsItem.AWP},
```

Comment:
Default this for CT and T to be not receiving an awp (not sure whether this is where you would make that change or not)